examples/json/lib/functions.dart

```diff
@@ -26,7 +24,19 @@ class GreetingResponse {
 
   GreetingResponse({this.salutation, this.name});
 
+  factory GreetingResponse.fromJson(Map<String, dynamic> json) =>
```

Comment:
README should get updated or else we should remove the "The full code is shown below" part